sumologic/sumologic_cmffgp.go

```diff
@@ -0,0 +1,181 @@
+package sumologic
+
+import (
+	"encoding/json"
+	"fmt"
+	"log"
+)
+
+func (s *Client) GetCmfFgp(targetType string, targetId string) (*CmfFgpResponse, error) {
```

Comment:
nit: I would suggest removing "cmf" keyword from the code. "CMF" is an internal term and I would prefer to not leak in the public code.

Reply:
I would keep "CMF" term. Because, there will be one form of FGP showing up in future.